modular_skyrat/master_files/code/modules/client/preferences/mutant_parts.dm

```diff
@@ -68,18 +68,34 @@
 /datum/preference/toggle/eye_emissives/proc/is_allowed(datum/preferences/preferences)
 	return preferences.read_preference(/datum/preference/toggle/allow_emissives)
 
-// Body Markings
+// Body Markings - This isn't used anymore and thus I'm making it not do anything.
```

Comment:
Then remove it fully from the code?

Reply:
I didn't want to remove it yet, just in the off-chance that anything came up during the test-merges. I'll be removing it later on.

Also, having it removed in a subsequent PR will make it easier to reference in the future.